dftimewolf/cli/dftimewolf_recipes.py

```diff
@@ -164,6 +164,11 @@ def ParseArguments(self, arguments):
     self._AddRecipeOptions(argument_parser)
 
     self._command_line_options = argument_parser.parse_args(arguments)
+
+    if not hasattr(self._command_line_options, 'recipe'):
```

Comment:
what if recipe is set but None? Maybe:

```
if not getattr(self._command_line_options, 'recipe', None):
```

Reply:
Interesting, I was pretty sure the default return value for `getattr` if no third argument was provided was already None. I should revisit other codebases 😅. Thanks!

Reply:
It likely is, but I prefer it to be specific

Reply:
Ack, thanks for the clarification.